src/app_service/common/types.nim

```diff
@@ -84,9 +84,10 @@ proc initShard*(cluster: int = -1, index: int = -1): Shard =
   result.cluster = cluster
   result.index = index
 
-# ToDo: Will be streamlined to single TokenType under https://github.com/status-im/status-desktop/pull/12654/files
-type NewTokenType* {.pure.} = enum
+type TokenType* {.pure.} = enum
```

Comment:
I was thinking about how to avoid growing these catch-all `types` files. One way could be to follow the domain-driven design and not mix different topics (messaging with wallet). Does it make sense to have this into a `blockchain_types.nim` and the current `types.nim` into `messaging_types.nim`. Will this bring some clarity to the code for a newcomer?

Reply:
@stefandunca the reasoning for this is that eventually community tokens and wallet tokens are "tokens" and we are treating them even more so after the are listing the community tokens with account assets. hope that makes sense